gabbi/httpclient.py

```diff
@@ -55,21 +54,21 @@ def request(self, absolute_uri, method, body, headers, redirect):
 
 
 class VerboseHttp(Http):
-    """A subclass of Http that verbosely reports on activity.
+    """A subclass of ``Http`` which verbosely reports on activity.
 
     If the output is a tty or ``GABBI_FORCE_COLOR`` is set in the
     environment, then output will be colorized according to ``COLORMAP``.
 
     Output can include request and response headers, request and
-    response body content (if of a printable content-type), or both.
+    response body content (if of a printable content type), or both.
```

Comment:
If you specifically mean the HTTP header, backticks and capitalization would help clarify that.

Reply:
No, I mean the content type of the response, which will often be described by the header, but I don't mean the header itself. The hyphen was probably an incorrect(?) effort to try to mollify your concerns with my use of hyphens.

Reply:
heh My rule of thumb is: Hyphenate if it's a multi-word modifier (see?) - cf. [front end vs. front-end developer](https://css-tricks.com/poll-results-front-end-front-end/#article-header-id-0). Essentially, the hyphen acts like parentheses there, thus improving readabilty.